### DISCUSSION TITLE
Switch to RC100H results in max Heating Temp

### DISCUSSION THREAD

**Discussion by Ed1954** [emsesp/EMS-ESP32#2486] [General]
If i switch RC100H on, the Heating Temp goes to max.  and room influenze are stiil at 0. This was not happened in older Versions.

- [x] Searched the issue in [issues](https://github.com/emsesp/EMS-ESP32/issues)
- [x] Searched the issue in [discussions](https://github.com/emsesp/EMS-ESP32/discussions)
- [x] Searched the issue in the [docs](https://docs.emsesp.org/Troubleshooting/)
- [x] Searched the issue in the [chat](https://discord.gg/3J3GgnzpyT)

network |  
network | "WiFi"
hostname | "ems-esp"
RSSI | -65
WIFIReconnects | 0
BSSID | "set"
TxPowerSetting | 0
staticIP | true
lowBandwidth | false
disableSleep | true
enableMDNS | true
enableCORS | false
APProvisionMode | "disconnected"
APSecurity | "wpa2"
APSSID | "ems-esp"
ntp |  
NTPStatus | "connected"
enabled | true
server | "time.google.com"
tzLabel | "Europe/Amsterdam"
mqtt |  
MQTTStatus | "connected"
MQTTPublishes | 51752
MQTTQueued | 0
MQTTPublishFails | 0
MQTTReconnects | 0
enabled | true
clientID | "esp32-463bdaec"
keepAlive | 60
cleanSession | false
entityFormat | 0
base | "ems-esp"
discoveryPrefix | "homeassistant"
discoveryType | 0
nestedFormat | 1
haEnabled | true
mqttQos | 0
mqttRetain | false
publishTimeHeartbeat | 60
publishTimeBoiler | 10
publishTimeThermostat | 10
publishTimeSolar | 10
publishTimeMixer | 10
publishTimeWater | 10
publishTimeOther | 10
publishTimeSensor | 10
publishSingle | false
publish2command | false
sendResponse | false
syslog |  
enabled | false
sensor |  
temperatureSensors | 0
temperatureSensorReads | 0
temperatureSensorFails | 0
api |  
APICalls | 5
APIFails | 0
bus |  
busStatus | "connected"
busProtocol | "Buderus"
busTelegramsReceived | 366999
busReads | 94679
busWrites | 48
busIncompleteTelegrams | 0
busReadsFailed | 0
busWritesFailed | 0
busRxLineQuality | 100
busTxLineQuality | 100
settings |  
boardProfile | "S32S3"
locale | "de"
txMode | 1
emsBusID | 11
showerTimer | false
showerMinDuration | 180
showerAlert | false
hideLed | false
noTokenApi | false
readonlyMode | false
fahrenheit | false
dallasParasite | false
boolFormat | 1
boolDashboard | 1
enumFormat | 1
analogEnabled | false
telnetEnabled | true
maxWebLogBuffer | 50
webLogBuffer | 9
modbusEnabled | false
forceHeatingOff | false
developerMode | false
devices |  
0 |  
type | "boiler"
name | "ACU-MH/Condens 5000i, Greenstar 8000/GC9800IW, GB192i*2"
deviceID | "0x08"
productID | 195
brand | "Buderus"
version | "05.06"
entities | 85
handlersReceived | "0xBF 0xC2 0x14 0x15 0xD1 0xE3 0xE4 0xE5 0xE9 0x2E 0x3B 0x04"
handlersFetched | "0xE6 0xEA 0x28"
handlersPending | "0x10 0x11 0x1C 0x18 0x19 0x1A 0x35 0x16 0x33 0x34 0x26 0x2A"
handlersIgnored | "0x02E0 0x02EA 0xD7 0xD8 0x02D6 0xF7 0x17 0x36 0x04B4"
1 |  
type | "thermostat"
name | "BC400-FO/UI800, BC400"
deviceID | "0x10"
productID | 4
brand | ""
version | "49.08"
entities | 49
handlersReceived | "0x06 0x02BA 0x02BB 0x02BC 0x0267"
handlersFetched | "0x02A5 0x02B9 0x02AF 0x029B 0x02CC 0x0291 0x0292 0x0293 0x0294 0x02F5 0x02F6 0x023A 0x0240"
handlersPending | "0xA3 0xA2 0x12 0x13 0x0471 0x02A6 0x02B0 0x029C 0x0472 0x02A7 0x02B1 0x029D 0x0473 0x02A8 0x02B2 0x029E 0x0474 0x02A9 0x02BD 0x02B3 0x029F 0x0475 0x02AA 0x02BE 0x02B4 0x02A0 0x0476 0x02AB 0x02BF 0x02B5 0x02A1 0x0477 0x02AC 0x02C0 0x02B6 0x02A2 0x0478 0x0467 0x02CE 0x0468 0x02D0 0x0469 0x02D2 0x046A 0x031B 0x031D 0x031E 0xBB 0x023E 0x0269"
handlersIgnored | "0xC4 0x047B 0xF7 0x31 0xBF 0xF6 0xF9 0x3B 0x10AC"
2 |  
type | "thermostat"
name | "RC100H, CR10H"
deviceID | "0x38"
productID | 200
brand | ""
version | "40.04"
entities | 3
handlersReceived | "0x042B 0x047B"
handlersPending | "0x0273 0x0A6A"
handlersIgnored | "0xF7"
3 |  
type | "gateway"
name | "MX300/WiFi module"
deviceID | "0x48"
productID | 252
brand | "Buderus"
version | "08.02"
entities | 0
handlersIgnored | "0xF9 0x2E 0x3B 0x2040 0xE5 0xE9 0x0CBD 0x0CC3 0x0DE4 0x0DE6 0x0DEF"

### TO REPRODUCE

Just switch RC100H on and off

![image](https://github.com/user-attachments/assets/52e73eb3-327f-49c3-ae3e-11b267964d69)
![image](https://github.com/user-attachments/assets/94a40ca4-07bb-4d02-ab13-dfcad2bb4cce)

**Comment by Ed1954** [comment]
Forgot to mention Logamax plus GB182i.2 not GB192i.2 as detected

**Comment by proddy** [comment]
anyone else reporting the same problem?

**Comment by sjheinzweiler** [comment]
Hi, same problem ems-esp firmware 2.8.1 : RC310/RC3*0, Moduline 3000/1010H, CW400, Sense II, HPC410 (Thermostat)
Emulated RC100H works as ON: permanently max target  ems-esp Thermostat hc1 Target flow temperature and only / OFF: target =0

<img width="2160" height="1620" alt="image" src="https://github.com/user-attachments/assets/25db5263-76ce-4d52-81ac-40d82b1d09b5" />

as you can see in the top right. Changes of room and selected temperature did not have any influence on real target flow right diagram below

i start RC100H ~ 15:15 (before I use RC200 is also shown in diagramm)

My system Hybrid Buderus Logano plus KBH192i (Gas and Heatpump) MX400 newest code 13.00.02 

system is an eternal flame .... 😟

second question:

In discord is advised to set room influence to 1...
Above is written set it to O
Is there any summary where this is explained?

<img width="1361" height="225" alt="image" src="https://github.com/user-attachments/assets/50164598-8c69-40c6-a4a6-c8e5eacb9b07" />


thank you for help

cheers Stef
